src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,162 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public boolean add(T value) {
-        return false;
+        addToEnd(value);
```

Comment:
```suggestion
        addToTail(value);
```
let's keep naming consistent

Reply:
fixed